rules/integrations/google_workspace/google_workspace_alert_center_promotion.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2023/01/15"
+integration = ["google_workspace"]
+maturity = "production"
+min_stack_comments = "Google Workspace feature only present in 8.4+ stack versions"
+min_stack_version = "8.4.0"
+updated_date = "2023/01/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the occurrence of a security alert from the Google Workspace alerts center. Google Workspace's security alert
+center provides an overview of actionable alerts that may be affecting an organization's domain. An alert is a warning
+of a potential security issue that Google has detected.
+"""
+false_positives = [
+    """
+    To tune this rule, add exceptions to exclude any google_workspace.alert.type which should not trigger this rule.
+    """,
+    "For additional tuning, severity exceptions for google_workspace.alert.metadata.severity can be added.",
+]
+from = "now-130m"
+index = ["filebeat-*", "logs-google_workspace*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Forwarded Google Workspace Security Alert"
+note = """## Setup
+
+## Triage and analysis
+
+This is a promotion rule for Google Workspace security events, which are alertable events per the vendor.
+Consult vendor documentation on interpreting specific events.
+"""
+references = ["https://workspace.google.com/products/admin/alert-center/"]
+risk_score = 73
+rule_id = "f1a6d0f4-95b8-11ed-9517-f661ea17fbcc"
+rule_name_override = "google_workspace.alert.type"
+severity = "high"
```

Comment:
I wonder if there is a way to preserve the origin severity of the alert : 

![image](https://user-images.githubusercontent.com/64742097/212751876-e29eb639-1366-45dc-93ae-b05ce7473bcd.png)

Reply:
Good point. I checked existing rules and Kibana and noticed there was a `SeverityMapping` dataclass. I added the mapping according to Google Workspace's documentation which states alerts are only low, medium or high. The value itself in ECS is capitalized however. I then re-imported the new rule and triggered it again and got the desired results where the severity is now low instead of high.

Risk score is currently unable to be mapped since Google Workspace does not supply a risk score, only severity. Additionally, I attempted to map the severity field to risk score, however, it only accepts integer values.

<img width="709" alt="Screen Shot 2023-01-16 at 9 08 49 PM" src="https://user-images.githubusercontent.com/99630311/212796035-841bda3d-a4f3-466c-96ae-b27793fee860.png">

<img width="808" alt="Screen Shot 2023-01-16 at 9 20 03 PM" src="https://user-images.githubusercontent.com/99630311/212796055-521dc8d2-585a-4e4d-a84c-39b0143e1a26.png">